docs/v2-system-addon/data_dictionary.md

```diff
@@ -191,6 +193,7 @@ and losing focus. | :one:
 | `user_prefs` | [Required] The encoded integer of user's preferences. | :one: & :four:
 | `is_prerendered` | [Required] A boolean to signify whether the page is prerendered or not | :one:
 | `is_preloaded` | [Required] A boolean to signify whether the page is preloaded or not | :one:
+| `icon_type` | Either ("tippytop", "rich_icon", "screenshot_with_icon", "screenshot", "no_image") | :one:
```

Comment:
Let's add a `[Optional]` as well as a note to say that it's only available in the "click" event ping of Top sites.

Reply:
It should go out with any kind of action you take on a topsite (click, block, delete, pin etc).